doc/architectural/CBMC-developer-guide.md

```diff
@@ -70,12 +255,12 @@ In the top level of `src` there are only a few files:
 
 * `doxygen.cfg`:   The config file for doxygen.cfg
 
-## `doc/`
+## `doc/` ##
```

Comment:
FWIW I think this is unnecessary in markdown

Reply:
Yeah, I discovered it yesterday and thought it might make the titles stand out more when looking at the source